client/components/web-preview/index.jsx

```diff
@@ -250,9 +225,46 @@ const WebPreview = React.createClass( {
 			</RootChild>
 		);
 	}
-} );
+}
+
+WebPreview.propTypes = {
```

Comment:
Granted it's a stage 1 feature, but FYI we've enabled [class static properties](https://github.com/jeffmo/es-class-public-fields) so you can declare `propTypes` and `defaultProps` within the class itself (e.g. `static propTypes = { /* ... */ };`